### PR TITLE
Release Argentina-AFIP-invoices 1.0.0

### PR DESCRIPTION
**Pull request by fgampel**

### CODE REVIEW

**Comment by keksa** [comment]
@fgampel Do you really need to include the tools directory with the plugin distribution? From what I see, you're not actually using anything from there in the actual plugin. If that is true, could you please remove the directory?

File: plugins/argentina-afip-invoices/src/classes/Argentinacae/claves/cwkey.key [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (keksa) [inline plugins/argentina-afip-invoices/src/classes/Argentinacae/claves/cwkey.key]: why is there a private key included? are you sure it should be here?

Comment (fgampel) [reply plugins/argentina-afip-invoices/src/classes/Argentinacae/claves/cwkey.key]: Hi @keksa this private key is used when customer decides to use the "debug function" included in the UCRM settings... With Debug, the plugin uses this private key to get "test results" instead to requiere a valid new certificate....

**Comment by fgampel** [comment]
Hi @keksa the tools directory is intended to a help to the final user... in there you have information on how to generate and enable it inside "Argentina AFIP Webservices"... is not needed by the plugin, but many user will need this info for creating their own certificates...

**Comment by keksa** [comment]
@fgampel Alright, however it's mainly the `tools/ssl/include/openssl/` directory I have problem with. I see no need to include the actual openssl C code in the plugin distribution.

**Comment by fgampel** [comment]
Hi @keksa the executable is there because the final user has to generate an privatekey, using this SSL, we can not include it, but the customer should start by himself to search the way to generate it.
That's why i included it, to give a complete solution to the final user. 
But, if you want i can move it to a document folder like Docs or something like that if it is better for you...

**Comment by keksa** [comment]
@fgampel I have no problem with the executable, but with the actual source code. All the `*.h` and `*.c` files in the `plugins/argentina-afip-invoices/tools/ssl/include/openssl/` directory. For example https://github.com/Ubiquiti-App/UCRM-plugins/pull/111/files#diff-888cacbb77e7f3496759561a9a75dda1

**Comment by fgampel** [comment]
@keksa oh! you're right... i didn't realize of the source codes there...  we can remove it... please tell me, can you remove it? or should i remove it and create a new pull request?

**Comment by keksa** [comment]
@fgampel You just remove the files, commit the changes and push it to the same branch in your repo (i.e. `argentina-afip-invoices` in your fork), this PR will automatically update.

**Comment by fgampel** [comment]
Hi @keksa I did what you requested to me, please check if i did it well. Kind Regards

**Comment by keksa** [comment]
Hello @fgampel, you did it correctly, thank you! We'll let you know in case we find any other issues.

In the meantime you can take a look at the failing automatic tests for this plugin. If you click on the details link at the bottom of this PR next to the "All checks have failed", you can see what's wrong (or just go to https://travis-ci.org/Ubiquiti-App/UCRM-plugins/builds/495464995).

In this case, you have wrong plugin URL in your manifest.json file, you have "https://github.com/Ubiquiti-App/UCRM-plugins" there, but it should be "https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/argentina-afip-invoices". This lets users in UCRM click on the link and see the README.md file of your plugin to get more information.

There is also `plugins.json` file out of date. We can handle this after this PR gets merged, but you can fix this easily just by running `php generate-json.php > plugins.json` in the root directory of your forked repository (i.e. not in the directory of your plugin, but in the top-most directory of this repository).

**Comment by fgampel** [comment]
Hi @keksa Manifest is updated now with the right URL.
But when i try to run in Git Bash on the top directory php generate-json.php > plugins.json i get an error saying stdout is not a tty, and my plugin.json file get completly empty.

**Pull request merged**